utils/local_remote_settings/__main__.py

```diff
@@ -0,0 +1,661 @@
+"""
+Runs models for use within Firefox via local Remote Settings.
+
+The artifacts will be saved to: ./data/artifacts
+
+To run the local models:
+ * Open Firefox (debug builds don't seem to work)
+ * Run the following from the web console.
+
+    ChromeUtils.importESModule("resource://services-settings/remote-settings.sys.mjs")
+        .RemoteSettings("translations-wasm").verifySignature = false;
+    ChromeUtils.importESModule("resource://services-settings/remote-settings.sys.mjs")
+        .RemoteSettings("translations-models").verifySignature = false;
```

Comment:
> praise

This is a lot nicer than having to modify the source code!

Thanks for updating the `--help` menu.